packages/wrangler/src/__tests__/d1/migrate.test.ts

```diff
@@ -196,12 +201,10 @@ Ok to create /tmp/my-migrations-go-here?`,
 Your database may not be available to serve requests during the migration, continue?`,
 				result: true,
 			});
-			//if we get to this point, wrangler knows the account_id
+
 			await expect(
 				runWrangler("d1 migrations apply db --remote")
-			).rejects.toThrowError(
-				`request to https://api.cloudflare.com/client/v4/accounts/nx01/d1/database/xxxx/backup failed`
-			);
+			).rejects.toThrowErrorMatchingInlineSnapshot(`"Failed to fetch"`);
```

Comment:
This doesn't seem right?

Reply:
This looks weird, but preserves the current behaviour of the test. https://github.com/cloudflare/workers-sdk/blob/1c8f981711bcc9d0430b47bf584460e4865f1505/packages/wrangler/src/__tests__/d1/migrate.test.ts#L172-L173 ensures that the `/backup` call is made with the right account ID, and then throws a network error. Previously, the test relied on no mock being defined, and was using a snapshot of the request failure to validate that the account ID was correct (`nx01`)